### ISSUE TITLE
preg_match fails when string has "/" in it

### ISSUE THREAD

**Issue by akizor** [Loilo/Fuse#5]
`loilo/fuse/src/Bitap/regex_search.php: 7`

Example trace
```
  Fuse\Bitap\regex_search("R. Aazyol", "Atelier Richard Woodman Burbridge/Harrods", " +")
  vendor/loilo/fuse/src/Bitap/Bitap.php : 63
```

**Comment by Loilo** [comment]
Thanks for reporting, I'll take a look. 👍

**Comment by Loilo** [comment]
Okay, I simply forgot to escape slashes explicitely. Previously piped the search pattern through `preg_quote` which obviously doesn't catch slashes.

Should now be fixed in 3.2.3.

Thanks again for reporting. :)
